P/Pitchjx/Compat.toml

```diff
@@ -1,5 +1,11 @@
-[0]
+["0-0.0.1"]
 DataFrames = "0.17.1-0"
 EzXML = "0.9.1-0"
 HTTP = "0.8-0"
 julia = "0.7-1"
+
+["0.0.2-0"]
+DataFrames = "0.17.1-0.17"
```

Comment:
Doesn't work with DataFrames 0.18?

Reply:
Just tried and it worked well. I'll update thie version.

Reply:
If it works with 0.17 you can support both.

Reply:
Just fixed `Project.toml` as `DataFrames =  ">= 0.17.1"`.